src/D2L.Bmx/WriteHandler.cs

```diff
@@ -41,13 +41,10 @@ BmxConfig config
 			}
 		}
 
-		var credentialsFile = new SharedCredentialsFile();
-		if( !string.IsNullOrEmpty( output ) ) {
-			if( !Path.IsPathRooted( output ) ) {
-				output = "./" + output;
-			}
-			credentialsFile = new SharedCredentialsFile( output );
+		if( !string.IsNullOrEmpty( output ) && !Path.IsPathRooted( output ) ) {
+			output = "./" + output;
 		}
+		var credentialsFile = new SharedCredentialsFile( output );
```

Comment:
Previously, we create two `SharedCredentialsFile` objects if `output` is not empty, which is wasteful.

The `SharedCredentialsFile(string)` constructor accepts a null parameter.
See https://github.com/aws/aws-sdk-net/blob/bb43d8be152ba92f4aeeb0038b8d69bd2a243b18/sdk/src/Core/Amazon.Runtime/CredentialManagement/SharedCredentialsFile.cs#L217-L252